public/index.html

```diff
@@ -43,14 +46,21 @@
 
     inPage = setup(window, document, "script", "sajari", "//address.js");
     inPage("config", {
-      project: "etstestest",       // Set this to your project.
-      collection: "trent-kiwi", // Set this to your collection.
+      project: "sajariptyltd",       // Set this to your project.
+      collection: "sajari-com", // Set this to your collection.
       values: { resultsPerPage: "10", "q": getUrlParam("q") },
       attachSearchBox: document.getElementById("search-box"),
       attachSearchResponse: document.getElementById("search-response"),
       pipeline: "website", // Run the website pipeline.
       searchBoxPlaceHolder: "Type to search", // Placeholder text in search box.
       results: { showImages: false }, // Show images in results.
+      input: {
+        instant: false,
+        autocomplete: true,
+        autoFocus: true, // Whether to focus the element on load
+        placeholder: "Search", // Placeholder text in the input element
+        amount: 5
```

Comment:
`numResults` ? Should be more consistent

Reply:
`numSuggestions`